src/main/java/com/mraof/minestuck/block/SulfurBubbleBlock.java

```diff
@@ -0,0 +1,21 @@
+package com.mraof.minestuck.block;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.block.TallGrassBlock;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class SulfurBubbleBlock extends TallGrassBlock
```

Comment:
TallGrassBlock is not a suitable class to extend. BushBlock is not the best option either but ideally we will need something more custom (perhaps customized class can come after PR)

Reply:
from rose's branch but understood